### PR TITLE
Sync tests for practice exercise ``protein translation``

### PR DESCRIPTION
**Pull request by manumafe98**
# pull request

This issue addresses: [#2388](https://github.com/exercism/java/issues/2388)

The goal is to sync the tests of the phone number protein translation

Reviewer Resources:

[Track Policies](https://github.com/exercism/java/blob/main/POLICIES.md#event-checklist)

### CODE REVIEW

**Comment by manumafe98** [comment]
I'm  going to update the reference resolution on this one as well

File: exercises/practice/protein-translation/src/test/java/ProteinTranslatorTest.java [hunk 1]
```diff
@@ -149,7 +150,54 @@ public void testTranslationStopsIfStopCodonInMiddle1() {
...
         assertThat(proteinTranslator.translate("UGGUGUUAUUAAUGGUUU"))
-            .containsExactly("Tryptophan", "Cysteine", "Tyrosine");
+                .containsExactly("Tryptophan", "Cysteine", "Tyrosine");
     }
 
+    @Ignore("Remove to run test")
+    @Test
+    public void testEmptyRnaSequenceResultInNoproteins() {
+        assertThat(proteinTranslator.translate("")).isEmpty();
+    }
```
Comment (sanderploegsma) [inline exercises/practice/protein-translation/src/test/java/ProteinTranslatorTest.java:160]: Note: The order of the tests in the canonical data may be ordered like that on purpose, for instance to cover basic test cases before more elaborate ones. This should aid students in their implementation.

To that end, we try to keep the order of the tests in the Java track in the same way as they are ordered in the canonical data. That would mean that this test for an empty sequence becomes the first test. Note that this also means that it shouldn't have an `@Ignore` tag and that the existing `testTranslationOfRnaToProteinList` method should be `@Ignore`d.

**Review by sanderploegsma — APPROVED** [review]
Changes look good! I left one comment regarding the order of the tests compared to the canonical data, could you have a look at that?

File: exercises/practice/protein-translation/src/test/java/ProteinTranslatorTest.java (hunk 1 above)
Comment (manumafe98) [reply exercises/practice/protein-translation/src/test/java/ProteinTranslatorTest.java]: Oh okey good to know! I updated the order to match the canonical data!

**Review by sanderploegsma — APPROVED** [review]

**Pull request merged**